kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/preprocess/SinkRecordFilter.java

```diff
@@ -0,0 +1,103 @@
+package com.wepay.kafka.connect.bigquery.preprocess;
+
+import com.jayway.jsonpath.*;
+import com.wepay.kafka.connect.bigquery.config.BigQuerySinkConfig;
+import com.wepay.kafka.connect.bigquery.convert.RecordConverter;
+import org.apache.kafka.common.config.ConfigException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class SinkRecordFilter {
+    private BigQuerySinkConfig bigQueryConfig;
+    private RecordConverter recordConverter;
+
+    private String configType = null;
+    private String configString = null;
+    private JsonPath jsonPathCondition = null;
+    private static final Logger logger = Logger.getLogger(SinkRecordFilter.class);
+
+    public SinkRecordFilter(BigQuerySinkConfig bigQueryConfig) {
+        this.bigQueryConfig = bigQueryConfig;
+        this.recordConverter = bigQueryConfig.getRecordConverter();
+
+        if (this.bigQueryConfig.getString(this.bigQueryConfig.INCLUDE_CONDITION_CONFIG) != null) {
+            this.configType = this.bigQueryConfig.INCLUDE_CONDITION_CONFIG;
+            this.configString = this.bigQueryConfig.getString(this.bigQueryConfig.INCLUDE_CONDITION_CONFIG);
+
+        } else if (this.bigQueryConfig.getString(bigQueryConfig.EXCLUDE_CONDITION_CONFIG) != null) {
+            this.configType = this.bigQueryConfig.EXCLUDE_CONDITION_CONFIG;
+            this.configString = this.bigQueryConfig.getString(bigQueryConfig.EXCLUDE_CONDITION_CONFIG);
+            this.jsonPathCondition = JsonPath.compile(this.configString);
+        }
+
+        if (configType != null) {
```

Comment:
looks like we only support one kind of config (either include or exclude). It might be useful to provide both? I don't think the documentation was very clear on that.